src/check-for-ami.js

```diff
@@ -39,7 +39,7 @@ module.exports = async function (ec2, ami) {
   }
 
   if (result.data.Images[0].ImageId === ami) {
-    return true;
+    return result.data.Images[0];
```

Comment:
yep, this should be fine.  Going from `true` to `Object` should pass all the same truthiness checks that exist elsewhere in the codebase